docker/build/runtime.x86_64.dockerfile.sample

```diff
@@ -72,5 +72,5 @@ RUN apt-get -y update \
     && rm -rf /var/lib/apt/lists/*
 
 # Restore
-RUN bash /opt/apollo/installers/install_geo_adjustment.sh us
+RUN bash /opt/apollo/installers/install_geo_adjustment.sh ${GEOLOC}
```

Comment:
Hi Martin, there is a nuance here. Geo-specific settings was already done in the `install_minimal_environment.sh` step to speed up Docker build process.
And it was reset to `US` in the last step  so that people all over the world will see the same settings. 

And I would like to here suggestions and opinions from different view points.

Reply:
AHA, so it was intentionally, will remove this change.